frontend/components/shared/Footer.tsx

```diff
@@ -0,0 +1,205 @@
+import { Image, Link } from '@nextui-org/react'
+
+const contactMethods = [
+  {
+    id: 0,
+    name: 'Customer Support',
+    link: '#',
+  },
+  {
+    id: 1,
+    name: 'Service Guarantee',
+    link: '#',
+  },
+  {
+    id: 2,
+    name: 'Website Feedback',
+    link: '#',
+  },
+]
+
+const companyInfo = [
+  {
+    id: 0,
+    name: 'About Us',
+    link: '#',
+  },
+  {
+    id: 1,
+    name: 'Terms and Condition',
+    link: '#',
+  },
+  {
+    id: 2,
+    name: 'Privacy and Cookies',
+    link: '#',
+  },
+]
+
+const partnerMethods = [
+  {
+    id: 0,
+    name: 'Partner Portal',
+    link: '#',
+  },
+  {
+    id: 1,
+    name: 'Affiliate Program',
+    link: '#',
+  },
+]
+
+const paymentImage = [
+  {
+    id: 0,
+    text: 'Visa',
+    src: '/payment_logos/visa.png',
+  },
+  {
+    id: 1,
+    text: 'Mastercard',
+    src: '/payment_logos/mastercard.png',
+  },
+  {
+    id: 2,
+    text: 'G-Cash',
+    src: '/payment_logos/Gcash.png',
+  },
+  {
+    id: 3,
+    text: 'GrabPay',
+    src: '/payment_logos/GrabPay.png',
+  },
+  {
+    id: 4,
+    text: 'PayMaya',
+    src: '/payment_logos/PayMaya.png',
+  },
+  {
+    id: 5,
+    text: 'BPI',
+    src: '/payment_logos/BPI.png',
+  },
+  {
+    id: 6,
+    text: 'Security Bank',
+    src: '/payment_logos/SecurityBank.png',
+  },
+  {
+    id: 7,
+    text: 'AUB',
+    src: '/payment_logos/AUB.png',
+  },
+  {
+    id: 8,
+    text: 'BillEase',
+    src: '/payment_logos/BillEase.png',
+  },
+  {
+    id: 9,
+    text: 'CoinsPH',
+    src: '/payment_logos/CoinsPH.png',
+  },
+  {
+    id: 10,
+    text: '7-11',
+    src: '/payment_logos/7_eleven.png',
+  },
+  {
+    id: 11,
+    text: 'Cebuana Lhuiller',
+    src: '/payment_logos/CebuanaLL.png',
+  },
+  {
+    id: 12,
+    text: 'M Lhuiller',
+    src: '/payment_logos/M_LL.png',
+  },
+  {
+    id: 13,
+    text: 'Landbank',
+    src: '/payment_logos/Landbank.jpg',
+  },
+]
+
+export const Footer = () => {
+  return (
+    <div className="flex justify-center bg-gray-100 h-auto static bottom-0 border-t-1 border-solid border-gray-300 w-full">
+      <div className="flex flex-col justify-center items-center w-full">
+        <div className="flex-wrap container px-6 lg:px-20 xl:px-35 2xl:px-14 items-start flex justify-center w-full py-7">
+          <div className="flex flex-nowrap flex-1 justify-center text-xs mx-auto flex-shrink-0 max-w-md">
+            <div className="flex flex-col justify-start p-4">
+              <span className="font-bold text-base pb-1.5">Contact Us</span>
+              <ul>
+                {contactMethods.map((contact) => (
```

Comment:
You don't have to manually put ids in the arrays. The .map() method provides the array's index and can be used as the key.

Try something like:
.map(contact, i) i is the index